theme/themes/pastanaga/modules/popup.overrides

```diff
@@ -7,3 +7,63 @@
     vertical-align: middle;
   }
 }
+
+
+.dropdown-popup-trigger{
```

Comment:
I think this file needs prettier for formatting. Usually there's a space before curlies.